integration/src/main/java/scripts/testscripts/ValidateReferencedResources.java

```diff
@@ -25,21 +28,34 @@
 
 public class ValidateReferencedResources extends DataRepoTestScriptBase {
 
+  // liam.dragonmaw@test.firecloud.org
   private TestUserSpecification secondUser;
+  // elijah.thunderlord@test.firecloud.org
+  private TestUserSpecification thirdUser;
```

Comment:
Commenting test user names here is a little misleading since that's specified in the test configuration file and liable to change.

To make things easier to follow, can you name these test users and their API client objects by function (e.g. test user objects like `fileReader` and `noAccessUser`, client objects like `fileReaderApi`, `noAccessUserApi`)